### ISSUE TITLE
The accuracy of pretrained model

### ISSUE THREAD

**Issue by WhuRan** [clovaai/deep-text-recognition-benchmark#206]
I run the test.py with the pretrained released model and get the results as below:

IIIT5k_3000: 87.367	SVT: 87.326	IC03_860: 95.116	IC03_867: 94.694IC13_857: 92.999	IC13_1015: 92.217	IC15_1811: 78.244	IC15_2077: 75.390	SVTP: 80.155	CUTE80: 74.216	total_accuracy: 85.249	averaged_infer_time: 1.641	# parameters: 49.555

The accuracy score is lower than your paper, why?

**Comment by ku21fan** [comment]
Hello, 

I guess that you used the pretrained model TRBA (TPS-ResNet-BiLSTM-Attn.pth).

Actually, that total_accuracy score is higher than our paper (our paper's total_accuracy is 84.0, see [our paper's supplementary, Table 8](https://arxiv.org/abs/1904.01906)).

As you said, some datasets (IIIT5k, SVT, IC13_857, IC13_1015) are lower than our paper, while other datasets (IC03_860, IC03_867, IC15_1811, IC15_2077, SP, CT) are better than our paper. 

The possible reasons are below. 
1) In the paper, we have run five trials with different initialization random seeds and have averaged their accuracies. 
We uploaded only one model which has better than average total_accuracy 84.0.

2) In this repository, we used [batch balance training](https://github.com/clovaai/deep-text-recognition-benchmark/blob/master/dataset.py#L17), which usually results in better accuracy.

3) The different versions of libraries such as PyTorch version could result in slightly different accuracy.

Hope it helps :)

Best
